diagnostics-extension/src/__tests__/telemetry.service.spec.ts

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Unit tests for telemetry.service
+ */
+
+import {
+  TelemetryService,
+  TelemetryServiceProvider,
+} from '../telemetry.service';
+import * as fakeData from '../fake_telemetry.data';
+import { DpslTypes } from '../dpsl.types';
+
+describe('should return instance of FakeTelemetryService', () => {
+  const telemetryService: TelemetryService =
+    TelemetryServiceProvider.getTelemetryService();
```

Comment:
Let's create an issue to track faking/mocking the returned object here, when the actual implementation exists.

Reply:
As per the slack discussion.

Reply:
https://github.com/MahmoudAGawad/cros-diag-app/issues/27